.github/workflows/docs-preview-links.yml

```diff
@@ -1,26 +1,19 @@
-name: Docs Preview Links
+name: docs-preview
 
 on:
   pull_request_target:
     types: [opened]
-    paths:
-      - '**.asciidoc'
```

Comment:
You can consider keeping the above condition if you don't want the action to post a comment on non doc changes.

Reply:
Good point, I've re-added it via c9fe55a91e15cb2c389ea179da4584d72eb92f2c